src/Smart.Design.Library.Showcase.WebApp/Pages/Templates/Subscription.cshtml

```diff
@@ -0,0 +1,237 @@
+
+                    <h3 class="c-h2 u-spacer-bottom-xl">Adresse Officielle</h3>
+
+                    <div class="o-grid">
+                        <div class="o-grid-col-6">
+                            <smart-formgroup label="Rue">
+                                <smart-input id="subscriptionStreet" name="subscriptionStreet" placeholder="rue du joli bois"></smart-input>
+                            </smart-formgroup>
+                        </div>
+                        <div class="o-grid-col-3">
+                            <smart-formgroup label="Numéro">
+                                <smart-input data-pristine-required-message="Error Message" id="number" name="number" placeholder="44"></smart-input>
+                            </smart-formgroup>
+                        </div>
+                        <div class="o-grid-col-3">
+                            <smart-formgroup label="Boîte">
+                                <smart-input id="subscriptionBox" name="subscriptionBox" placeholder="3"></smart-input>
+                            </smart-formgroup>
+                        </div>
+                    </div>
+
+                    <div class="o-grid">
+                        <div class="o-grid-col-4">
+                            <smart-formgroup label="Code postal">
+                                <smart-input id="subscriptionZip" name="subscriptionZip" placeholder="4000"></smart-input>
+                            </smart-formgroup>
+                        </div>
+                        <div class="o-grid-col-8">
+                            <smart-formgroup label="Ville">
+                                <smart-input id="subscriptionCity" name="subscriptionCity" placeholder="Liège"></smart-input>
+                            </smart-formgroup>
+                        </div>
+                    </div>
+
+                    <smart-formgroup label="Pays">
+                        <smart-select id="subscriptionCountry" name="subscriptionCountry">
+                            <option>Belgique</option>
+                            <option>Espagne</option>
+                            <option>Neptune</option>
+                        </smart-select>
+                    </smart-formgroup>
+
+                    <smart-hr></smart-hr>
+
+                    <h2 class="c-h2">Adresse Courrier</h2>
+
+                    <div class="u-spacer-bottom">
+                        <smart-radio id="subscriptionSameAddress" name="subscriptionAddressChoice" label="Mon adresse courrier est la même que mon adresse officielle" value="same_adress" checked="true"></smart-radio>
+                    </div>
+                    <div class="u-spacer-bottom-xl">
+                        <smart-radio id="subscriptionAnotherAddress" name="subscriptionAddressChoice" label="Mon adresse courrier est différente de mon adresse officielle" value="same_address" checked="false"></smart-radio>
+                    </div>
```

Comment:
Pour le moment ça ne fonctionne pas ce module. Si je clique que on adresse courrier est différente, je n'ai pas le formulaire adressse courrier

Reply:
To be done : https://smartjira.atlassian.net/browse/PAIL-70